src/lib/utils.ts

```diff
@@ -12,4 +12,17 @@ export const pathnameWithLocale = (locale: Locale, pathname: string) => (
 
 export const bpsToPercent = (bps: number) => (
   `${(bps / 100).toFixed(2)}%`
-)
\ No newline at end of file
+)
+
+export const formatTime = (time: number | undefined) => {
+  if (time || time === 0) {
+    const hours = Math.floor(time / 3600)
+    const minutes = Math.floor((time % 3600) / 60)
+    const seconds = Math.floor(time % 60)
+    return `${String(hours).padStart(2, '0')}:${String(minutes).padStart(
+      2,
+      '0'
+    )}:${String(seconds).padStart(2, '0')}`
+  }
+  return '00:00:00'
+}
```

Comment:
can you create a new lib?
- rule: only export one function per lib file
- actual: this file is exporting 2, different, library functions
- suggestion: create a new lib: `src/lib/formatTime.ts`